packages/css/src/components/icon/README.md

```diff
@@ -29,3 +29,7 @@ Icons are aligned to the left of the text by default and vertically centred to t
 Icons use the same text levels as all typographic components to determine their size.
 This ensures easy alignment between icons and text.
 [Refer to the typography documentation for more information](/docs/docs-design-guidelines-typography--docs).
+
+## Overview
+
+[You can find an overview of the available icons here](/docs/docs-assets-icons--docs).
```

Comment:
Not very important, but the space between this and the Usage section seems too small:

![image](https://github.com/Amsterdam/design-system/assets/7290629/fcd8e020-e4df-4a23-b68a-d31dc2599ade)

Reply:
‘Usage’ shouldn’t be a `h1`. Making it a `h2` will add margin at the top.